packages/search/package.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "@automattic/search",
+	"version": "1.0.0-alpha.3",
+	"description": "Automattic Search",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"sideEffects": [
+		"*.css",
+		"*.scss"
+	],
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/search"
+	},
+	"publishConfig": {
+		"access": "public"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"dist",
+		"src"
+	],
+	"types": "dist/types",
+	"dependencies": {
+		"@automattic/viewport": "^1.0.0",
+		"@babel/runtime": "^7.11.1",
+		"@wordpress/components": "^10.0.5",
+		"@wordpress/i18n": "^3.14.0",
+		"@wordpress/icons": "^2.4.0",
+		"classnames": "^2.2.6",
+		"lodash": "^4.17.15"
+	},
+	"peerDependencies": {
+		"react": "^16.8",
+		"react-dom": "^16.8"
+	},
+	"devDependencies": {
+		"@storybook/addon-actions": "^5.3.18",
+		"enzyme": "^3.11.0"
```

Comment:
Since we don't have any tests yet, it seems we can remove `enzyme` from the list of dependencies, no?